.circleci/config.yml

```diff
@@ -387,18 +387,21 @@ jobs:
         description: The golang arch
         type: string
         default: amd64
-    executor: golang
+    executor: # we are using linux/arm64 and linux/amd64 vm's for build
+      name: vm-<< parameters.arch >>
+    environment:
+      GOPATH: /home/circleci/.go-kuma-go
     steps:
+    - install_build_tools:
+        go_arch: <<parameters.arch>>
     - checkout
     - restore_cache:
         keys:
         # prefer the exact match
         - golang-go.mod-{{ .Branch }}-{{ checksum "go.sum" }}
     - run:
         name: Build all Kuma binaries (such as, kumactl, kuma-cp, kuma-dp, kuma-prometheus-sd)
-        command: make build
-    - setup_remote_docker:
-        version: *docker_version
+        command: GOARCH=<<parameters.arch>> make build
```

Comment:
Any reason not to put that straight in `environment`?

Reply:
I've just realized that because we are building them on specific archs we don't need this env, it's going to be read correctly from the system. Removed it.